Source/ACE.Server/WorldObjects/Entity/CreatureVital.cs

```diff
@@ -92,12 +92,9 @@ public uint MaxValue
         {
             get
             {
-                uint total = Base;
+                var attr = AttributeFormula.GetFormula(creature, Vital, true);
 
-                var multiplier = creature.EnchantmentManager.GetVitalMod_Multiplier(this);
```

Comment:
this should still be getting the enchantments here

```
var attr = AttributeFormula.GetFormula(creature, Vital, true);

                uint total = StartingValue + Ranks + attr;

                var multiplier = creature.EnchantmentManager.GetVitalMod_Multiplier(this);
                var additives = creature.EnchantmentManager.GetVitalMod_Additives(this);

                total = (uint)Math.Round(total * multiplier + additives);
```

Reply:
what do those account for? are they not included in the AttributeForumula? my char vitals were properly updating with endurance/willpower without them

Reply:
looks like these may be cantrips, essence glutton, life giver, etc...

Reply:
EnchantmentManager would be all of the enchantments

GetVitalMod_Multiplier would be things like asheron's benediction
GetVitalMod_Additive would be things like essence glutton